clients/python/inspr/client.py

```diff
@@ -0,0 +1,57 @@
+import os
+import json
+import sys
+from flask import Flask, request, url_for
+from http import HTTPStatus
+from .rest import *
+from types import FunctionType
+
+SIDECAR_READ_PORT = "INSPR_SCCLIENT_READ_PORT"
+SIDECAR_WRITE_PORT = "INSPR_LBSIDECAR_WRITE_PORT"
+
+class Client:
+    def __init__(self) -> None:
+        self.readPort = os.getenv(SIDECAR_READ_PORT)
+        self.writeAddress = "http://localhost:" + str(os.getenv(SIDECAR_WRITE_PORT))
+        self.app = Flask(__name__)
+
+    
+    def writeMessage(self, channel:str, msg) -> None:
+        msgBody = {
+            "data": msg
+        }
+        jsonObj = json.dumps(msgBody, indent=4)
+        try:
+            sendPostRequest(self.writeAddress + "/" + channel, jsonObj)
+        except:
+            print("Error while trying to write message")
+            raise ValueError
+
+    
+    def handleChannel(self, channel:str, handleFunc:FunctionType) -> None:
+        print("channel =", channel)
```

Comment:
printing every time handle is called? seem useful for debugging but not desired when using the client